.eslintrc.js

```diff
@@ -13,7 +13,7 @@ module.exports = {
     rules: {
         "no-console": "off",
         "@typescript-eslint/explicit-function-return-type": ["off"],
-        "@typescript-eslint/no-unused-vars": ["error", { argsIgnorePattern: "^_" }],
+        "@typescript-eslint/no-unused-vars": "off",
```

Comment:
Do we need to switch-off the error on unused vars? What for? In case it's absolutely needed, couldn't we just warn the developer in these situations?

Reply:
Carried from early stages of development
_/no-unused-vars_ is set as before: ["error", { argsIgnorePattern: "^_" }]